packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -56,71 +87,166 @@ export const pageQuery = graphql`
 
 ### About Cards
 
-Cards are a specifically-styled container that group relatable content and actions.
-Card is an extremely flexible container that does not require specific components inside of it,
-allowing an implementer to compose as needed, but elements such as Title, Body,
-and Button or Anchor are commonly used. See Examples for common Card patterns.
-A Card does not currently handle any interactive events such as hover, click or focus,
-though children composed inside of it may commonly have event handlers.
+Cards are specifically-styled containers that group relatable content and actions. Card is an extremely 
+flexible container that does not require specific components inside of it, allowing an implementer to 
+compose as needed, but elements such as Heading, Text, and Button or Anchor are commonly used.
```

Comment:
- Break up the sentence to: "...does not require specific components inside of it. You can compose the card as you need it, but elements such as Heading, Text, Button, and Anchor are commonly used."
- Link to each of those components.